packages/components/src/Toggle/Toggle.scss

```diff
@@ -0,0 +1,75 @@
+// all
+$toggle-height: 20px;
+
+$toggle-track-height: 10px;
```

Comment:
I think your toogle track is longer and thinner than the one in the guidelines. You should check it with Maxime.